frodo.js

```diff
@@ -2,133 +2,132 @@
  * Extras Plugin
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.